src/Library/demos/Spin Button/main.blp

```diff
@@ -0,0 +1,19 @@
+using Gtk 4.0;
+
+Box welcome {
+  orientation: vertical;
+  valign: center;
+  halign: center;
+
+  Label {
+    label: "Spin Button";
+    margin-bottom: 30;
+    styles ["title-1"]
+  }
+
+  Button open_spinbox{
+    label: "Open";
+    styles["pill"]
+  }
+
+}
```

Comment:
the links to online resources are missing

Reply:
please also add the HIG link like on other entries

https://developer.gnome.org/hig/patterns/controls/spin-buttons.html